src/main/java/org/icgc/argo/program_service/services/ego/EgoService.java

```diff
@@ -79,17 +79,6 @@ public EgoService(
     this.invitationRepository = invitationRepository;
   }
 
-  public EgoService(@NonNull ProgramEgoGroupRepository programEgoGroupRepository,
-    @NonNull ProgramConverter programConverter,
-    @NonNull EgoClient restClient,
-    @NonNull JoinProgramInviteRepository invitationRepository,
-    RSAPublicKey key) {
```

Comment:
They are slightly different. This constructor has a RSAPublicKey parameter though it does not handle the key correctly. 
 Is this constructor no longer needed?

Reply:
The constructor is no longer needed, RSAPublicKey is removed from EgoService to EgoSecurity.